lib/ExpensiMark.js

```diff
@@ -160,7 +160,7 @@ export default class ExpensiMark {
             {
                 name: 'newline',
                 regex: /\n/g,
-                replacement: '<br>',
+                replacement: '<br />',
```

Comment:
Will this affect ordinary messages as well as edited ones? Have we tested what this does in the "normal" situation of ppl sending each other messages?

Reply:
I ask bc the tests in the OP only mention  editing an already-sent message

Reply:
Good question. I've looked into this here - https://github.com/Expensify/App/issues/7268#issuecomment-1019659872

Basically, even if we pass `Line1<br>Line2` to the API, the server returns `Line1<br />Line2` on fresh login, and that's what is stored in Onyx.

I couldn't find any side effects that'll occur after implementing this change. I can't speak of any internal repos that use `Expensimark.replace()` though

Reply:
awesome, thx @rushatgabhane